Userland/Libraries/LibGUI/EmojiInputDialog.cpp

```diff
@@ -153,6 +153,11 @@ EmojiInputDialog::EmojiInputDialog(Window* parent_window)
     m_search_box->on_change = [this]() {
         update_displayed_emoji();
     };
+
+    m_search_box->on_return_pressed = [this]() {
+        update_displayed_emoji();
```

Comment:
Do we need to call `update_displayed_emoji` here? All filters would have been applied already by the search / category actions, no?